Simulator.lua

```diff
@@ -1117,6 +1118,8 @@ function Simulator:injectApi(a_ApiDesc)
 		assert(type(prevIndex) == "function")  -- We don't support table-based __index yet (but can be done)
 		return prevIndex(a_Table, a_SymbolName)
 	end
+	
```

Comment:
You'll need to change the `__index` function slightly, too - it needs to not only return the value, but also set it in the sandbox, so that the class' definition with prefilled symbols doesn't get GCed and replaced with a new one.